src/E1_coldflow/E1_coldflow.cpp

```diff
@@ -105,26 +100,25 @@ void loop() {
     int i = 0;
     while (RFSerial.available()) {
       command[i] = RFSerial.read();
-      Serial.print(command[i]);
       i++;
     }
+    Serial.write(command, i);
```

Comment:
This is just for debugging right? Delete before merge?

Reply:
yeah. I'll incorporate the DEBUG flag.